components/tyk-bootstrap/values.yaml

```diff
@@ -52,6 +52,9 @@ global:
     useSecretName: ""
     devPortal: tyk-dev-portal-conf
 
+  # The default name for the sa is k8s-bootstrap-role
+  serviceAccountName: "k8s-bootstrap-role"
```

Comment:
why do we need to make this field global? Can't we move it to the component level?

Reply:
It was included in global field because it is central to the deployment in the component chart. Why do you think it shouldn't be there if i may ask?

Reply:
so, if the field is located global section, any umbrella chart using `tyk-bootstrap` subchart will access to this field. Assume `tyk-chart` is in use. So, in the global field we have `serviceAccountName` which will configure Tyk Dashboard, Tyk Gateway, Tyk Pump, Tyk Enterprise Portal and Tyk Bootstrap. For all of these components, the users have to use 1 service account. However, for instance tyk-bootstrap needs separate service account as it belongs to chart hooks. That's why i thought moving serviceAccount fields to component level. If users want to use custom service account, they can specify it; otherwise, k8s will pick up the `default` SA

Reply:
Okay. This makes sense. I can remove it from the global value then.